main.py

```diff
@@ -1,45 +1,42 @@
-import requests
 import json
 from flask import Flask, request
-app = Flask(__name__)
-
-REQUIRED_LABELS = ['ready for review']
+from PullRequest import PullRequest
+from config import REQUIRED_LABELS_ALL, REQUIRED_LABELS_ANY, BANNED_LABELS
 
-# config = {
-#     "all": ["label1", "label2"],
-#     "not": ["label3"],
-#     "any": ["label4", "label5"]
-# }
+app = Flask(__name__)
 
 
-@app.route('/', methods=["POST"])
+@app.route('/', methods=["POST", "GET"])
```

Comment:
When do we need to `GET` from this endpoint?

Reply:
This is needed for testing. You can't load this URL in a web browser unless GETs are allowed

Reply:
you should be able to use `curl` to post to a url, eg:
```shell
curl -X POST localhost:5000 -d '{"json_data": "here"}'
```